front/app/modules/commercial/id_vienna_saml/components/ViennaIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const ViennaIcon = () => {
+  return (
+    <svg width="22" height="22" style={{ flexShrink: '0', marginTop: '2px' }}>
```

Comment:
using styled-components here for some reason doesn't work. Lucas and me fiddled around with it, but in the end settled to use inline CSS here as the icon is used only in one place.